src/Build/Construction/ProjectRootElement.cs

```diff
@@ -712,6 +720,18 @@ internal ProjectExtensionsElement ProjectExtensions
         internal string LastDirtyReason
             => _dirtyReason == null ? null : String.Format(CultureInfo.InvariantCulture, _dirtyReason, _dirtyParameter);
 
+        /// <summary>
+        /// Initialize an in-memory, empty ProjectRootElement instance that CANNOT be saved later.
```

Comment:
[typo] If this means that a new empty instance is created in-memory, then the comma is incorrect here. Otherwise, the bit before the comma looks incomplete.